workers/unity/Packages/com.improbable.gdk.gameobjectcreation/GameObjectCreatorFromMetadata.cs

```diff
@@ -67,15 +67,15 @@ public GameObject OnEntityCreated(SpatialOSEntity entity)
             }
 
             var gameObject = Object.Instantiate(prefab, position, Quaternion.identity);
-            gameObject.name = $"{prefab.name}(SpatialOS: {entity.SpatialOSEntityId}, worker {workerType}";
+            gameObject.name = $"{prefab.name}(SpatialOS: {entity.SpatialOSEntityId}, Worker: {workerType}";
             return gameObject;
         }
 
-        public void OnEntityRemoved(SpatialOSEntity entity, GameObject linkedGameObject)
+        public void OnEntityRemoved(SpatialEntityId entityId, GameObject gameObject)
```

Comment:
Mentioned somewhere else but this should use the same arg names as the interface does. I don't really see anything wrong with linked being in the name, but I don't really mind one way or the other.